tests/zugferd211/Tests/BuilderTest.php

```diff
@@ -197,4 +201,183 @@ public function testBuildXRechnungExample(): void
+        $sellerTradeParty->globalID[] = Id::create('4000001123452', '0088');
+        $sellerTradeParty->name = 'Lieferant GmbH';
+        $sellerTradeParty->definedTradeContact = new TradeContact();
+        $sellerTradeParty->definedTradeContact->personName = 'Max Mustermann';
+        $sellerTradeParty->definedTradeContact->departmentName = 'Muster-Einkauf';
+        $sellerTradeParty->definedTradeContact->telephoneUniversalCommunication = new UniversalCommunication();
+        $sellerTradeParty->definedTradeContact->telephoneUniversalCommunication->completeNumber = '+49891234567';
+        $sellerTradeParty->definedTradeContact->emailURIUniversalCommunication = new UniversalCommunication();
+        $sellerTradeParty->definedTradeContact->emailURIUniversalCommunication->uriid = Id::create('Max@Mustermann.de');
+
+        $sellerTradeParty->postalTradeAddress = new TradeAddress();
+        $sellerTradeParty->postalTradeAddress->postcode = '80333';
+        $sellerTradeParty->postalTradeAddress->lineOne = 'Lieferantenstraße 20';
+        $sellerTradeParty->postalTradeAddress->city = 'München';
+        $sellerTradeParty->postalTradeAddress->countryCode = 'DE';
+
+        $sellerTradeParty->taxRegistrations[] = TaxRegistration::create('201/113/40209', 'FC');
+        $sellerTradeParty->taxRegistrations[] = TaxRegistration::create('DE123456789', 'VA');
+
+        $invoice->supplyChainTradeTransaction->applicableHeaderTradeAgreement->buyerTradeParty = $buyerTradeParty = new TradeParty();
+        $buyerTradeParty->id = Id::create('GE2020211');
+        $buyerTradeParty->name = 'Kunden AG Mitte';
+
+        $buyerTradeParty->postalTradeAddress = new TradeAddress();
+        $buyerTradeParty->postalTradeAddress->postcode = '69876';
+        $buyerTradeParty->postalTradeAddress->lineOne = 'Kundenstraße 15';
+        $buyerTradeParty->postalTradeAddress->city = 'Frankfurt';
+        $buyerTradeParty->postalTradeAddress->countryCode = 'DE';
+
+        $invoice->supplyChainTradeTransaction->applicableHeaderTradeAgreement->specifiedProcuringProject = SpecifiedProcuringProject::create('1234', 'Projekt');
+
+        $invoice->supplyChainTradeTransaction->applicableHeaderTradeDelivery = new HeaderTradeDelivery();
+        $invoice->supplyChainTradeTransaction->applicableHeaderTradeDelivery->shipToTradeParty = $buyerTradeParty;
+        $invoice->supplyChainTradeTransaction->applicableHeaderTradeDelivery->chainEvent = new SupplyChainEvent();
+        $invoice->supplyChainTradeTransaction->applicableHeaderTradeDelivery->chainEvent->date = DateTime::create('102', '20180305');
+
+        $invoice->supplyChainTradeTransaction->applicableHeaderTradeDelivery->deliveryNoteReferencedDocument = ReferencedDocument::create('123456');
+        $invoice->supplyChainTradeTransaction->applicableHeaderTradeDelivery->deliveryNoteReferencedDocument->formattedIssueDateTime = QualifiedDateTime::create('102', '20180305');
```

Comment:
What do you use for validation?
I tried this https://www.zugferd-community.net/en/dashboard/validation and it comes back with more errors:
![image](https://user-images.githubusercontent.com/13612429/131484690-1ef1d3e7-2f4f-4f3d-bfb8-470c93d3b751.png)


And then I realized I forgot to change the profile to `urn:cen.eu:en16931:2017#conformant#urn:zugferd.de:2p1:extended`

![image](https://user-images.githubusercontent.com/13612429/131485037-4bc7316c-6100-494c-8fc3-72f440194d7d.png)

Any thoughts on using this profile?

Reply:
I used https://www.epoconsulting.com/erechnung-sap/xrechnung-validator but I think the underlying validator is always the same.

I think the extended profile is correct for this test case. I'm not sure if X-Rechnung actually allows any of the additional elements.

Reply:
I think you are correct. If I use the X-Rechnung profile then it will fail with all the extended elements.

I changed the profile to the extended zugferd so.  The validator you linked fails with it as the validator seems to only support the standard X-Rechnung profiles. The validator I linked passes. I hade to change the shipping cost to 0 as the validator was failing with a sum check.